Podfile.lock

```diff
@@ -24,15 +24,6 @@ DEPENDENCIES:
   - PromiseKit (~> 4.0)
   - Quick (~> 0.10.0)
 
-SPEC REPOS:
-  https://github.com/CocoaPods/Specs.git:
-    - BEMCheckBox
-    - JSONModel
-    - MBProgressHUD
-    - Nimble
-    - PromiseKit
-    - Quick
-
```

Comment:
Why did this change?

Reply:
Yeah this might be an auto change (when running pod install) on the pairing machine. I'll reverted back.